src/js/base/stories/RelativeTime.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { RelativeTime } from "../RelativeTime";
+
+const currentDate = new Date();
+
+export default {
+    title: "base/RelativeTime",
+    component: RelativeTime
+};
+
+const Template = args => <RelativeTime {...args} />;
+
+export const currentRelativeTime = Template.bind({});
+
+currentRelativeTime.args = {
+    time: currentDate
+};
+
+export const futureRelativeTime = Template.bind({});
+
+futureRelativeTime.args = {
+    time: new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate() + 7)
+};
+
+export const pastRelativeTime = Template.bind({});
+
+pastRelativeTime.args = {
+    time: new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate() - 7)
+};
```

Comment:
The datetimes look like they are still shifting to me: 

![image](https://user-images.githubusercontent.com/59776400/200694683-9196b0ce-fc21-450a-a68b-667f4e66aa58.png)


I had assumed that this would be linked with when the the last preview was published, but then I probably would have expected it to have shifted by 5 days here and not one. Not fixed by refreshing.

Reply:
This behaves slightly unexpectedly but not incorrectly. The logic how it was with +7 days and -7 days display the in 6 days and 8 days ago. I updated the logic so that it always displays in 7 days and 7 days ago. But refreshing does reset it on my end both locally and in preview.

Reply:
Ahhh okay, I assumed that the 6/8 were from the time shift and not a result of the core logic